intuita-webview/src/codemodList/TreeView/index.tsx

```diff
@@ -14,14 +14,45 @@ import styles from './style.module.css';
 import cn from 'classnames';
 import { useProgressBar } from '../useProgressBar';
 import { VSCodeButton } from '@vscode/webview-ui-toolkit/react';
+import { SEARCH_QUERY_MIN_LENGTH } from '../../shared/SearchBar';
 
 type Props = Readonly<{
 	node: CodemodTreeNode;
 	autocompleteItems: string[];
 	openedIds: ReadonlySet<CodemodHash>;
 	focusedId: CodemodHash | null;
+	searchQuery: string;
 }>;
 
+export const containsSearchedCodemod = (
+	node: CodemodTreeNode,
+	searchQuery: string,
+	set: Set<CodemodHash>,
+): Set<CodemodHash> | null => {
```

Comment:
do we need to return the set if we're modifying it?
also, why are we modifying the set here? this is the data collector, am I right?